### ISSUE TITLE
Some more requests

### ISSUE THREAD

**Issue by AnyoneMouse** [fast4x/RiMusic#762]
## Description

- [ ] **1.** Filter option for playlists.
For example filter "downloaded" or "not downloaded" or "certain artist" or "other artist except certain one" or "certain length" or "favorited" or "not favorited" etc. songs.
#706 #621 

- [x] **2.** Pin 📌 playlists.

- [x] **3.** If possible add "most played playlist" sorting in library.

- [x] **4.** A random button at the top of "Artist" and "Album" tab. Meaning ability to open random "artist" and "album" from those tabs.

- [ ] **5.**Tabs sorting feature. Also/or show/hide tabs feature.

- [ ] **6.** Sort each playlist separately. #736 

### Probably not so good/important and just for my personal request:

- [x] **7.** Custom playlist cover. #380 

- [x] **8.** Toggle song length to see time left. I mean something like this: 
![InCollage_20240218_204832215](https://github.com/fast4x/RiMusic/assets/85461823/c3329b94-93aa-41e9-99b4-0db84db87dd6)

- [x] **9.** While playing multiple enqueued songs show how much time left to listen. Probably above the thumbnail in the center?

- [x] **10.** The feature of swiping down (while player is minimized) to stop playing song is kinda troublesome I think. So, I'd like if there was a enable/disable for this at least.

- [x] **11.** The menu button (three dotted button) was better placed before. Now it's almost in the way of "favorite" button.
(Probably because I keep the thumbnail size medium. But I checked even in big thumbnail size it's kinda hard to use.)
Probably just place it somewhere here… 
![Screenshot_20240218-225952](https://github.com/fast4x/RiMusic/assets/85461823/bc271282-456c-4275-8de0-371e84fddd7e)

~~- [ ] It could be bug but I'm guessing it's not. It's probably just a normal thing. Because I'm uncertain I didn't report it as bug…
I'll be showing this through screen record because I can't really explain it!
https://github.com/fast4x/RiMusic/assets/85461823/0bea9a1f-93f6-4acd-ad93-9fd69be182da
Do you get the problem?~~

~~- [ ] Is there any way that we could see artist playlists as well?
ig it's not really possible…and probably I'm not sure but probably "artists" doesn't even come from "YouTube channels"…sorry I was just mistaken…I'm striking through this as well~~

### Additional Context

Let me know if the way I wrote these requests is wrong! And sorry if it is. I'd like to do it properly. Thanks.

**Comment by fast4x** [comment]
`It could be bug but I'm guessing it's not. It's probably just a normal thing. Because I'm uncertain I didn't report it as bug…
I'll be showing this through screen record because I can't really explain it!
https://github.com/fast4x/RiMusic/assets/85461823/0bea9a1f-93f6-4acd-ad93-9fd69be182da
Do you get the problem?`

Yes, isn't a bug...

**Comment by fast4x** [comment]
No, that’s correct. That’s a lot of things so I’ll ask you a few details later if I need them.

**Comment by fast4x** [comment]
If you can help, you see it already from the things you have written. They are interesting things.

**Comment by fast4x** [comment]
Moved menu button in action bar, enable/disable by settings

**Comment by fast4x** [comment]
Added time left in player centered between start and end time

**Comment by fast4x** [comment]
> > Added time left in player centered between start and end time
> 
> I edited the issue and made checklists. Have I marked the right ones?

Yes

**Comment by fast4x** [comment]
Nice checkbox

**Comment by fast4x** [comment]
You can create a private task, where you put your ideas over time to keep track of them, I won’t use it as a feature request.

**Comment by fast4x** [comment]
Example, You can create a discussion on your behalf and put in your notes.

**Comment by fast4x** [comment]
https://github.com/fast4x/RiMusic/discussions/new/choose

**Comment by fast4x** [comment]
Exactly

**Comment by fast4x** [comment]
Is there any way that we could see artist playlists as well?
Also this explain better

**Comment by fast4x** [comment]
A random button at the top of "Artist" and "Album" tab. Meaning ability to open random "artist" and "album" from those tabs
Implemented, available from version 0.6.24

**Comment by fast4x** [comment]
Please explain better: Is there any way that we could see artist playlists as well?

**Comment by fast4x** [comment]
Also: Tabs sorting feature. Also/or show/hide tabs feature.
Tabs in home?

**Comment by fast4x** [comment]
> While playing multiple enqueued songs show how much time left to listen. Probably above the thumbnail in the center?

Implemented, available from version 0.6.25

**Comment by fast4x** [comment]
>  If possible add "most played playlist" sorting in library.

Implemented, available from version 0.6.25

**Comment by fast4x** [comment]
> show/hide tabs feature.

What exactly?

**Comment by fast4x** [comment]
Implemented pinned playlists in library, available from version 0.6.25

**Comment by PitchAbyss** [comment]
@fast4x hi sir it it possible to add "Added Dats" in sort under the playlist ?

**Comment by fast4x** [comment]
I think yes, create a feature request.

**Comment by PitchAbyss** [comment]
> I think yes, create a feature request.

thankyou sir
